contracts/test/contract.SL2RD.test.js

```diff
@@ -332,4 +464,46 @@ contract("SL2RD", (accounts) => {
         });
     }
   });
+
+  specify(
```

Comment:
Can we also design a test case specifically to address the creator reservation vs. community splits allocation case?